src/helpers/performance-report-analyser.helper.js

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
+import chalk from 'chalk';
+
+class PerformanceReportAnalyser {
+  getReport(fileName) {
```

Comment:
Actuallty I don't see a reason why we have to have a separate method for a one-liner ? It is not a public method (or at least shouldn't be). It should not be testable (what is a point of testing it in isolation as it does not bring anything - we want to test analyser not file opener) ?